ports/espressif/boards/adafruit_esp32s3_camera/mpconfigboard.mk

```diff
@@ -12,10 +12,13 @@ CIRCUITPY_ESP_FLASH_FREQ = 80m
 CIRCUITPY_ESP_PSRAM_SIZE = 2MB
 CIRCUITPY_ESP_PSRAM_MODE = qio
 CIRCUITPY_ESP_PSRAM_FREQ = 80m
-FLASH_SDKCONFIG = esp-idf-config/sdkconfig-4MB-1ota.defaults
+
+# No OTA partition: larger filesystem instead for lots of images.
```

Comment:
isn't it designed for a larger firmware, not more filesystem?

Reply:
yes, that is wrong, from an early misunderstanding of mine.